lib/mayaUsd/nodes/proxyAccessor.py

```diff
@@ -0,0 +1,286 @@
+#
+# Copyright 2020 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+This file contains a set of reference helper functions which can be used to test and discover
+new capabilities provided by proxy accessor on a proxy shape.
+
+The end goal is to leverage UFE to provide integration of mixed data model compute with
+workflows like parenting (USD prim to Maya DAG object or vice versa), keying, or constraining.
+For the time being, this set of helper functions can be used as a reference to understand
```

Comment:
This file was very useful to understand the entire PR and also for testing. Thanks!

Reply:
Happy to hear it!